repose-aggregator/services/service-client/api/src/main/java/org/openrepose/core/services/serviceclient/akka/AkkaServiceClient.java

```diff
@@ -29,6 +29,8 @@
  */
 public interface AkkaServiceClient {
 
+    void setConnectionPoolId(String connectionPoolId);
```

Comment:
So i was thinking about this. I think we revert all the changes to the akka service client itself, and stop making it a bean. Instead we make a factory bean. We wire the factory bean into the places that currently use the client. They ask the factory for a client for a given pool. then we can just add another arguement to the current client constructor and it works almost as is.